### ISSUE TITLE
ERROR: TestImageFont.test_textsize_equal

### ISSUE THREAD

**Issue by toabctl** [python-pillow/Pillow#1202]
Running test testsuite during building 2.7.0 on SLE11SP3 failed with the following error:

```
[   25s] ======================================================================
[   25s] FAIL: TestImageFont.test_textsize_equal
[   25s] ----------------------------------------------------------------------
[   25s] Traceback (most recent call last):
[   25s]   File "/usr/src/packages/BUILD/Pillow-2.7.0/Tests/test_imagefont.py", line 111, in test_textsize_equal
[   25s]     self.assert_image_equal(im, target_img)
[   25s]   File "/usr/src/packages/BUILD/Pillow-2.7.0/Tests/helper.py", line 81, in assert_image_equal
[   25s]     self.fail(msg or "got different content")
[   25s] AssertionError: got different content
[   25s] 
[   25s] ----------------------------------------------------------------------
[   25s] Ran 555 tests in 4.127s
[   25s] 
[   25s] FAILED (SKIP=95, failures=1)
[   25s] error: Bad exit status from /var/tmp/rpm-tmp.53836 (%check)
```

**Comment by toabctl** [comment]
I think assert_image_similar() should be used here.

**Comment by radarhere** [comment]
Just a note for others reading this that the test was added in #784.

If you find that assert_image_similar does pass the tests, then the fastest way to start this change would be to create a PR.

**Comment by radarhere** [comment]
You may also be interested in updating the 'Platform Support' section of Installation.rst.

**Comment by hugovk** [comment]
@toabctl Would you like to create a PR? Tips here: https://github.com/python-pillow/Pillow/blob/master/CONTRIBUTING.md

**Comment by toabctl** [comment]
PR is here: https://github.com/python-pillow/Pillow/pull/1210

**Comment by hugovk** [comment]
Fixed by #1210.
